cuda/solver/lower_trs_kernels.cu

```diff
@@ -88,16 +209,20 @@ void generate(std::shared_ptr<const CudaExecutor> exec,
 GKO_INSTANTIATE_FOR_EACH_VALUE_AND_INDEX_TYPE(
```

Comment:
You could also make the generate_kernel call conditional on the strategy type

Reply:
That seems sensible. I wanted to leave all that to an analysis-phase PR, but on second thought, this will come in handy when comparing total runtime.